SoftLayer/CLI/autoscale/create.py

```diff
@@ -0,0 +1,124 @@
+"""Order/Create a scale group."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+from SoftLayer import utils
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import formatting
+from SoftLayer.CLI import helpers
+from SoftLayer.managers.autoscale import AutoScaleManager
+
+
+@click.command()
+@click.option('--name', help="Scale group's name.")
+@click.option('--cooldown', type=click.INT,
+              help="The number of seconds this group will wait after lastActionDate before performing another action.")
+@click.option('--min', 'minimum', type=click.INT, help="Set the minimum number of guests")
+@click.option('--max', 'maximum', type=click.INT, help="Set the maximum number of guests")
+@click.option('--regional', type=click.INT,
+              help="The identifier of the regional group this scaling group is assigned to.")
+@click.option('--postinstall', '-i', help="Post-install script to download")
+@click.option('--os', '-o', help="OS install code. Tip: you can specify <OS>_LATEST")
```

Comment:
![image](https://user-images.githubusercontent.com/78806346/165851336-91bdec3f-9181-4fd8-85bf-9d75e8c55bc8.png)
Please add `required=True, prompt=True` to ensure virtual guest data input